tests/format-quick-bar.spec.ts

```diff
@@ -556,6 +563,8 @@ async function scrollToBottom(page: Page) {
     .evaluate(node =>
       node.scrollTo({ left: 0, top: 1000, behavior: 'smooth' })
     );
+  // TODO switch to `scrollend`
+  // See https://developer.chrome.com/en/blog/scrollend-a-new-javascript-event/
```

Comment:
Scrolled is a too latest feature. We need a polyfill if to do so

https://github.com/argyleink/scrollyfills/blob/main/src/scrollend.js

Reply:
This is a function for the e2e test only, we can use it when the test environment is ready.